### PR TITLE
adding personal keymaps

### PR DESCRIPTION
**Pull request by semicolonsnet**
## Description

Two personal keycaps

## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [ ] Keyboard (addition or update)
- [X] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Issues Fixed or Closed by This PR

* 

## Checklist

- [X] My code follows the code style of this project: [**C**](https://docs.qmk.fm/#/coding_conventions_c), [**Python**](https://docs.qmk.fm/#/coding_conventions_python)
- [X] I have read the [**PR Checklist** document](https://docs.qmk.fm/#/pr_checklist) and have made the appropriate changes.
- [ ] My change requires a change to the documentation.
- [X] I have updated the documentation accordingly.
- [X] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [X] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

File: keyboards/sixkeyboard/keymaps/semicolonsnet/keymap.c [hunk 1]
```diff
@@ -0,0 +1,26 @@
+#include QMK_KEYBOARD_H
```
Comment (drashna) [inline keyboards/sixkeyboard/keymaps/semicolonsnet/keymap.c:1]: Could you add a GPL2+ compatible license header here? 

For instance: 
https://github.com/qmk/qmk_firmware/blob/28ff51175b6d45ce5a8dbfe7ed6e7a12df3bc8a8/quantum/template/base/keyboard.c#L1-L15

File: keyboards/xd60/keymaps/semicolonsnet/keymap.c [hunk 2]
```diff
@@ -0,0 +1,11 @@
+#include QMK_KEYBOARD_H
```
Comment (drashna) [inline keyboards/xd60/keymaps/semicolonsnet/keymap.c:1]: Could you add a GPL2+ compatible license header here? 

For instance: 
https://github.com/qmk/qmk_firmware/blob/28ff51175b6d45ce5a8dbfe7ed6e7a12df3bc8a8/quantum/template/base/keyboard.c#L1-L15

File: keyboards/xd60/keymaps/semicolonsnet/keymap.c [hunk 3]
```diff
@@ -0,0 +1,11 @@
...
+	[1] = LAYOUT_all(KC_GRV, KC_F1, KC_F2, KC_F3, KC_F4, KC_F5, KC_F6, KC_F7, KC_F8, KC_F9, KC_F10, KC_F11, KC_F12, KC_F13, KC_F14, KC_TRNS, KC_TRNS, KC_UP, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_PSCR, KC_VOLU, KC_VOLD, KC_TRNS, KC_TRNS, KC_LEFT, KC_DOWN, KC_RGHT, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, LCAG(KC_SCLN), LCAG(KC_QUOT), KC_NO, KC_TRNS, KC_TRNS, KC_NO, LCAG(KC_Z), LCAG(KC_X), LCAG(KC_ASTR), LCAG(KC_V), LCAG(KC_B), LCAG(KC_N), LCAG(KC_M), LCAG(KC_COMM), KC_INS, KC_DEL, KC_TRNS, KC_PGUP, KC_INS, KC_TRNS, KC_TRNS, KC_TRNS, LCAG(KC_SPC), KC_TRNS, KC_TRNS, KC_HOME, KC_PGDN, KC_END)
+};
+
+// Loop
+void matrix_scan_user(void) {
+  // Empty
+};
```
Comment (drashna) [inline keyboards/xd60/keymaps/semicolonsnet/keymap.c:11]:
```suggestion

**Review by drashna — CHANGES_REQUESTED** [review]

**Review by semicolonsnet — COMMENTED** [review]
Requested changes made - thanks!

**Review by drashna — APPROVED** [review]

**Review by spidey3 — APPROVED** [review]

**Pull request merged**